planning/backlog.md

```diff
@@ -8,42 +8,35 @@ A collection of user stories you can choose from. Stories inside of each priorit
 
 are necessary for basic usability:
 
-- **user story name**: As a [type of user] I want to [do something] so that [I achieve some goal]
-  - Given [context] when [a specific action is performed] then [a set of consequences should occur]
-  - ...
-- **user story name**: As a [type of user] I want to [do something] so that [I achieve some goal]
-  - Given [context] when [a specific action is performed] then [a set of consequences should occur]
-  - ...
-- ...
+- **Website Visitor**: As a Website Visitor I want to have a clear navigation around the website   
+  - [ ] Given when I see the header and the footer, then links to the other parts of the website are provided
+- **Website Visitor**: As a Website Visitor I want to know whether I can use the solution on my computer 
+  - [ ] Given when I visit the website then I can see the list of languages supported, and the technical requirements
+- **Website Visitor**: As a Website Visitor I want to know how to install Loruki on my computer so that I could use it   
+  - [ ] Given when I visit the website then I can see the instruction on installing Loruki  
```

Comment:
@rayanejsilva, nice work making this more specific.  food for thought: if you find that your acceptance criteria (or user stories) could have an "and" in them, it might be time to split them into two

- Given when the user visits the home page then he can see brief information ont how to instal Loruki
- Given when the user visits the [other] page then he can see all the necessary instructions.

this will make it easier to convert into a checklist, and easier to think about as separate tasks

Reply:
@colevandersWands Thank you for your help. Splitting user stories is not so simple. 👩🏻‍💻😂😩